router/network.go

```
@@ -182,11 +190,12 @@ func (network *NetHandleT) SendPost(ctx context.Context, structData integrations
 			//Detecting content type of the respBody
 			contentTypeHeader = http.DetectContentType(respBody)
 		}
+		mediaType, _, _ := mime.ParseMediaType(contentTypeHeader)
```

we are ignoring error here